packages/plugin-doc-coverage/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@code-pushup/doc-coverage-plugin",
+  "version": "0.57.0",
+  "description": "Code PushUp plugin for tracking documentation coverage 📚",
+  "license": "MIT",
+  "homepage": "https://github.com/code-pushup/cli/tree/main/packages/plugin-doc-coverage#readme",
+  "bugs": {
+    "url": "https://github.com/code-pushup/cli/issues?q=is%3Aissue%20state%3Aopen%20type%3ABug%20label%3A\"🧩%20doc-coverage-plugin\""
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/code-pushup/cli.git",
+    "directory": "packages/plugin-doc-coverage"
+  },
+  "keywords": [
+    "CLI",
+    "Code PushUp",
+    "plugin",
+    "automation",
+    "developer tools",
+    "conformance",
+    "documentation coverage",
+    "documentation",
+    "docs",
+    "KPI tracking",
+    "automated feedback",
+    "regression guard",
+    "actionable feedback",
+    "audit",
+    "score monitoring"
```

Comment:
```suggestion
    "documentation coverage",
    "documentation quality",
    "docs completeness",
    "automated documentation checks",
    "coverage audit",
    "documentation conformance",
    "docs KPI tracking",
    "documentation feedback",
    "actionable documentation insights",
    "documentation regression guard",
    "documentation score monitoring",
    "developer documentation tools",
    "plugin for documentation coverage",
    "CLI documentation coverage",
    "Code PushUp documentation",
    "documentation audit"
```

Reply:
I tried to make them more focused, let me know what you think

Reply:
They seems good! I think on top of that I am gonna add documentation analyzer, it would not be the first time I have used that word to look for a plugin like this